server/app/routes/users/index.js

```diff
@@ -44,10 +45,10 @@ router.put('/:id', function (req, res, next) {
 });
 //combined*sv
 //get one user, their order and reviews
-router.get('/:id', function (req, res, next) {
+router.get('/member', function (req, res, next) {
```

Comment:
are we doing this for protection? i.e. are we obscuring the member id for security reasons?